esphome/components/toshiba/toshiba.cpp

```diff
@@ -439,6 +766,227 @@ void ToshibaClimate::transmit_rac_pt1411hwru_temp_(const bool cs_state, const bo
   }
 }
 
+void ToshibaClimate::transmit_ras_2819t_() {
+  // Handle swing mode transmission for RAS-2819T
+  // Note: RAS-2819T uses a toggle command, so we need to track state changes
+
+  // Check if ONLY swing mode changed (and no other climate parameters)
+  bool swing_changed = (this->swing_mode != this->last_swing_mode_);
+  bool mode_changed = (this->mode != this->last_mode_);
+  bool fan_changed = (this->fan_mode != this->last_fan_mode_);
+  bool temp_changed = (abs(this->target_temperature - this->last_target_temperature_) > 0.1f);
+
+  bool only_swing_changed = swing_changed && !mode_changed && !fan_changed && !temp_changed;
+
+  if (only_swing_changed) {
+    ESP_LOGI(TAG, "[TX] Swing mode changed from %s to %s, sending ONLY toggle command",
+             this->last_swing_mode_ == climate::CLIMATE_SWING_VERTICAL ? "VERTICAL" : "OFF",
+             this->swing_mode == climate::CLIMATE_SWING_VERTICAL ? "VERTICAL" : "OFF");
+
+    // Send ONLY swing toggle command (like the physical remote does)
+    auto swing_transmit = this->transmitter_->transmit();
+    auto *swing_data = swing_transmit.get_data();
+
+    // Convert toggle command to bytes for transmission
+    uint8_t swing_message[RAS_2819T_MESSAGE_LENGTH];
+    swing_message[0] = (RAS_2819T_SWING_TOGGLE >> 40) & 0xFF;
+    swing_message[1] = (RAS_2819T_SWING_TOGGLE >> 32) & 0xFF;
+    swing_message[2] = (RAS_2819T_SWING_TOGGLE >> 24) & 0xFF;
+    swing_message[3] = (RAS_2819T_SWING_TOGGLE >> 16) & 0xFF;
+    swing_message[4] = (RAS_2819T_SWING_TOGGLE >> 8) & 0xFF;
+    swing_message[5] = RAS_2819T_SWING_TOGGLE & 0xFF;
+
+    ESP_LOGD(TAG, "[TX] Swing toggle ONLY: %02X %02X %02X %02X %02X %02X", swing_message[0], swing_message[1],
+             swing_message[2], swing_message[3], swing_message[4], swing_message[5]);
+
+    // Use single packet transmission WITH repeat (like regular commands)
+    encode_(swing_data, swing_message, RAS_2819T_MESSAGE_LENGTH, 1);
```

Comment:
```suggestion
    this->encode_(swing_data, swing_message, RAS_2819T_MESSAGE_LENGTH, 1);
```
I see there are a whole bunch of calls to `encode_` throughout which are missing `this->` 🙈
Please add it anywhere it's missing. I realize it makes the PR bigger but it's an easy-to-understand change with essentially zero impact (aside from better compliance with our [codebase standards](https://developers.esphome.io/contributing/code/#c-code-style)).

Reply:
Addressed in 6b193fb10.